patches/amd-mainline/rocm-libraries/0024-hipFFT-and-rocFFT-test-integration-with-fftw3.patch

```diff
@@ -0,0 +1,77 @@
+From 2e4295e902270cf376ff0e6ff5163d68de7c60f3 Mon Sep 17 00:00:00 2001
+From: geomin12 <geomin12@amd.com>
+Date: Wed, 3 Sep 2025 08:29:35 -0700
+Subject: [PATCH 24/24] hipFFT and rocFFT  test integration with fftw3
+
+---
+ projects/hipfft/clients/tests/CMakeLists.txt |  4 ++--
+ projects/rocfft/clients/tests/CMakeLists.txt | 10 ++++++----
+ 2 files changed, 8 insertions(+), 6 deletions(-)
+
+diff --git a/projects/hipfft/clients/tests/CMakeLists.txt b/projects/hipfft/clients/tests/CMakeLists.txt
+index 7ce9b49987..0d73f984a7 100644
+--- a/projects/hipfft/clients/tests/CMakeLists.txt
++++ b/projects/hipfft/clients/tests/CMakeLists.txt
+@@ -181,7 +181,6 @@ foreach( target ${TEST_TARGETS} )
+   target_include_directories( ${target}
+     PRIVATE
+     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
+-    $<BUILD_INTERFACE:${FFTW_INCLUDE_DIRS}>
+     $<BUILD_INTERFACE:${hip_INCLUDE_DIRS}>
+     if( HIPFFT_BUILD_SCOPE )
+       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../library/include>
+@@ -191,7 +190,8 @@ foreach( target ${TEST_TARGETS} )
+   target_link_libraries( ${target}
+     PRIVATE
+     hip::hipfft
+-    ${FFTW_LIBRARIES}
++    FFTW3::fftw3
++    FFTW3::fftw3f
```

Comment:
@marbre how did we handle similar library vendoring in subprojects? I think having a `FFTW_LIBRARIES` variable is reasonable, if that variable is populated through supported mechanisms (e.g. `find_package`).